### PR TITLE
changes to episode 6, tasks 1-9

### PR DESCRIPTION
**Pull request by mallewellyn**
Largely re-writing motivation and methodology sections in line with tasks 1-9 of #119 

Particularly tries to clarify the differences between clustering and PCA/FA from the outset (even though they're completely different methods, I think a reader would benefit from this being apparent immediately).

Also moves the section on believing in clusters to after the method is explained - I think how to believe in a cluster is easier to understand if we know what clusters are fully first

### CODE REVIEW

File: _episodes_rmd/06-k-means.Rmd [hunk 1]
```diff
@@ -32,31 +32,84 @@ knitr_fig_path("08-")
...
+and this is discussed below. Once we have picked intitial points, we then follow 
+these two steps until appropriate clusters have been formed:
+
+1. Assign each data point to the cluster with the closest centroid
+2. Update centroid positions as the average of the points in that cluster
+
+We can see this process in action in this animation:
+
+```{r kmeans-animation, echo = FALSE, fig.cap="Cap", fig.alt="Alt"}
+knitr::include_graphics("../fig/kmeans.gif")
```
Comment (alanocallaghan) [inline _episodes_rmd/06-k-means.Rmd:88]:
```suggestion
knitr::include_graphics(here::here("fig/kmeans.gif"))
```

File: _episodes_rmd/06-k-means.Rmd [hunk 2]
```diff
@@ -32,31 +32,84 @@ knitr_fig_path("08-")
...
+and this is discussed below. Once we have picked intitial points, we then follow 
+these two steps until appropriate clusters have been formed:
+
+1. Assign each data point to the cluster with the closest centroid
+2. Update centroid positions as the average of the points in that cluster
+
+We can see this process in action in this animation:
+
+```{r kmeans-animation, echo = FALSE, fig.cap="Cap", fig.alt="Alt"}
+knitr::include_graphics(here::here("fig/kmeans.gif"))
```
Comment (alanocallaghan) [inline _episodes_rmd/06-k-means.Rmd:88]: knitr::include_graphics("../fig/kmeans.gif")

File: _episodes_rmd/06-k-means.Rmd [hunk 3]
```diff
@@ -32,31 +32,84 @@ knitr_fig_path("08-")
...
+so that we become more confident about the shape and size of the clusters.
+
+
+# What is K-means clustering?
+
+**K-means clustering** groups data points into a 
+user-defined number of distinct, non-overlapping clusters. 
+To create clusters of 'similar' data points, K-means 
+clustering creates clusters that minimise the 
+within-cluster variation adn thus the amount that
```
Comment (alanocallaghan) [inline _episodes_rmd/06-k-means.Rmd:72]: adn -> and

File: _episodes_rmd/06-k-means.Rmd [hunk 4]
```diff
@@ -32,31 +32,84 @@ knitr_fig_path("08-")
...
+The distance between data points within a cluster is 
+used as a measure of within-cluster variation.
+
+To carry out K-means clustering, we first pick $k$ initial points as centres or 
+"centroids" of our clusters. There are a few ways to choose these initial "centroids"
+and this is discussed below. Once we have picked intitial points, we then follow 
+these two steps until appropriate clusters have been formed:
+
+1. Assign each data point to the cluster with the closest centroid
+2. Update centroid positions as the average of the points in that cluster
```
Comment (alanocallaghan) [inline _episodes_rmd/06-k-means.Rmd:83]:
```suggestion
2. Update centroid positions as the average of the points in that cluster.
```

File: _episodes_rmd/06-k-means.Rmd [hunk 5]
```diff
@@ -32,31 +32,84 @@ knitr_fig_path("08-")
...
+
+We can see this process in action in this animation:
+
+```{r kmeans-animation, echo = FALSE, fig.cap="Cap", fig.alt="Alt"}
+knitr::include_graphics("../fig/kmeans.gif")
+```
+While K-means has some advantages over other clustering methods (easy to implement and
+to understand), it does have some disadvantages, namely difficulties in identifying
```
Comment (alanocallaghan) [inline _episodes_rmd/06-k-means.Rmd:91]: namely -> particularly

(just so we're not implying these are the only ones)

File: _episodes_rmd/06-k-means.Rmd [hunk 6]
```diff
@@ -32,31 +32,84 @@ knitr_fig_path("08-")
...
+2. Update centroid positions as the average of the points in that cluster
+
+We can see this process in action in this animation:
+
+```{r kmeans-animation, echo = FALSE, fig.cap="Cap", fig.alt="Alt"}
+knitr::include_graphics("../fig/kmeans.gif")
+```
+While K-means has some advantages over other clustering methods (easy to implement and
+to understand), it does have some disadvantages, namely difficulties in identifying 
+initial clusters which observations belong to and the need for the user to specifiy the
```
Comment (alanocallaghan) [inline _episodes_rmd/06-k-means.Rmd:92]: specifiy -> specify

**Review by alanocallaghan — CHANGES_REQUESTED** [review]
Looks great just minor problems

File: _episodes_rmd/06-k-means.Rmd (hunk 3 above)
Comment (mallewellyn) [reply _episodes_rmd/06-k-means.Rmd]: 7f7b8ff

File: _episodes_rmd/06-k-means.Rmd (hunk 5 above)
Comment (mallewellyn) [reply _episodes_rmd/06-k-means.Rmd]: ee8917a

File: _episodes_rmd/06-k-means.Rmd (hunk 6 above)
Comment (mallewellyn) [reply _episodes_rmd/06-k-means.Rmd]: 67a85aa

**Comment by mallewellyn** [comment]
All fixed above :)

**Review by alanocallaghan — APPROVED** [review]

**Comment by alanocallaghan** [comment]
Awesome thanks, great addition

**Pull request merged**